src/components/structures/auth/SetupEncryptionBody.tsx

```diff
@@ -112,19 +116,23 @@ export default class SetupEncryptionBody extends React.Component<IProps, IState>
         store.returnAfterSkip();
     };
 
-    private onResetClick = (ev: ButtonEvent): void => {
-        ev.preventDefault();
+    private onResetClick = (): void => {
+        const store = SetupEncryptionStore.sharedInstance();
         Modal.createDialog(ResetIdentityDialog, {
             onReset: () => {
                 // The user completed the reset process - close this dialog
                 this.props.onFinished();
                 const store = SetupEncryptionStore.sharedInstance();
                 store.done();
             },
-            variant: "confirm",
+            variant: store.lostKeys() ? "cant_confirm" : "confirm",
         });
     };
 
+    private onLogOutClick = (): void => {
```

Comment:
for consistency, we seem to use `logout` as per https://dictionary.cambridge.org/dictionary/english/logout rather than `log out` so O should not be capitalised here

Reply:
I had considered calling it `onSignOutClick`, but decided to use "log out" for consistency, since that term seemed to be used more in the code.  But there are places in the code that use "sign out", so maybe I will change it.  (Though I think that there is a case to be made about using the same terminology everywhere in the code for the same function, independent of what it says in the UI.  For example, it makes it easier to find places that deal with logging out by grepping for a single string.)

If I do change the function to `onSignOutClick`, should I also change the property name to `allowSignOut` rather than `allowLogout`?